tests/conftest.py

```diff
@@ -48,39 +80,252 @@ def swg_config(indexd_client): # noqa
     config.host = indexd_client.url
     config.username = indexd_client.auth[0]
     config.password = indexd_client.auth[1]
-    yield config
+    return config
+
+
+@pytest.fixture
```

Comment:
This feels suspicious. If the goal is to just run all the tests with either swg_config or swg_config_no_migrate? you can just provide a parameter to the existing swg_config fixture:
```
@pytest.fixture(params = [indexd_client.url, indexd_client_no_migrate.url])
def swg_config(request, indexd_client): # noqa
    config = swagger_client.Configuration()
    config.host = request.param
    config.username = indexd_client.auth[0]
    config.password = indexd_client.auth[1]
    return config
```
Let pytest do the lifting.

Reply:
Also, note the config is functionally scoped? I feel like it could be session scoped safely just because it is configuration and there's no finalizers/cleanup going on.

Reply:
bump

Reply:
Sorry I didn't get to this. Does this mean that every test that uses swg_config will run for both parameters? We don't want that. They each have distinct and separate use cases.

Reply:
mkay, there's definitely fixtures in here tho that are utilizing fixtures that aren't in the parameter list.. that will be confusing for somebody